automated install/basic-install.sh

```diff
@@ -57,6 +57,17 @@ Cloudflare (DNSSEC);1.1.1.1;1.0.0.1;2606:4700:4700::1111;2606:4700:4700::1001
 EOM
 )
 
+DNS_SERVERSv6ONLY=$(
```

Comment:
```suggestion
DNS_SERVERS_IPV6_ONLY=$(
```

Just a minor niggle from me. I long since abandoned IPv6 personally so have no otherwise useful comments on this PR ;)